tests/collections/collection.test.ts

```diff
@@ -591,28 +635,28 @@ describe(`StargateMongoose - ${testClient} Connection - collections.collection`,
       assert.strictEqual(deleteManyResp.acknowledged, true);
     });
     it('should throw an error when deleteMany finds more than 20 records', async () => {
-      let docList = Array.from({ length: 20 }, ()=>({"username": "id", "city" : "trichy"}));
+      let docList = Array.from({ length: 20 }, () => ({ "username": "id", "city": "trichy" }));
       docList.forEach((doc, index) => {
-        doc.username = doc.username+(index+1);
+        doc.username = doc.username + (index + 1);
       });
       const res = await collection.insertMany(docList);
       assert.strictEqual(res.insertedCount, 20);
       //insert next 20
-      let docListNextSet = Array.from({ length: 20 }, ()=>({username: "id", city : "trichy"}));
+      let docListNextSet = Array.from({ length: 20 }, () => ({ username: "id", city: "trichy" }));
```

Comment:
Amazing, I didn't know about this `Array.from()` syntax, learn something new every day. I usually do `Array(20).fill(0).map(() => ({ username: 'id' }))` but I like this approach better :+1: